content/en/_index.md

```diff
@@ -967,6 +967,17 @@ public static void exampleSnippet(String projectId, String filePath) {
 
 {{< content-tab header="PHP" >}}
 
+### Linting
+
+We follow the [PSR-1][psr-1] and [PSR-2][psr-2] community coding standards, with
+one notable exception:
+   
+1. The function names for sample are in `snake_case` rather than `camelCase`,
+   in order to differentiate them from class methods.
+   
+[psr-1]: https://www.php-fig.org/psr/psr-1/
+[psr-2]: https://www.php-fig.org/psr/psr-2/#1-overview
+   
```

Comment:
I think I was unclear. Please move this to line 882 with a new content-tabpane under the exising linting guideline.

Reply:
I'm not familiar with the syntax being used here, or how to preview the changes. I am also sure what you're asking. This is a very minor and low priority change and we have it documented internally so I am closing this instead.